src/app/common/base-test.helpers.spec.ts

```diff
@@ -19,6 +19,7 @@ export function emitClickOnElement<T>(fixture: ComponentFixture<T>, selector: st
 
 export function emitClickOnElementByDataTestId<T>(fixture: ComponentFixture<T>, id: string): void {
     const debugElement: MaybeNull<DebugElement> = fixture.debugElement.query(By.css(`[data-test-id="${id}"]`));
+    console.log(`[data-test-id="${id}"]`);
```

Comment:
Is it necessary?

Reply:
It's not, thank you!

Just want to mention this PR is still in Draft, I will clean it up :)